.circleci/config.yml

```diff
@@ -139,16 +139,6 @@ workflows:
             branches:
               ignore:
                 - master
-      - test-windows:
-          name: Windows Tests for Node v8 support
-          context: nodejs-install
-          node_version: "8.17.0"
```

Comment:
Regarding this, could you update also to a higher version? https://github.com/snyk/nodejs-lockfile-parser/blob/master/package.json#L25

Reply:
https://github.com/snyk/nodejs-lockfile-parser/pull/95